Gemfile

```diff
@@ -1,31 +1,31 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.1.3"
+ruby '3.1.3'
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/221355001-7bf37fea-8bde-4e82-83bd-c1c0144f9af3.png)

- [ ] Kindly remove this Live Demo section since you don't have a live link in your readme file :+1: to make your readme file descriptive and professional :blush: Once you will have your live link you can add this Live Demo Section with live link in your readme file :+1:

Reply:
done.